meilisearch-http/src/index/updates.rs

```diff
@@ -87,6 +87,28 @@ impl Settings<Checked> {
             _kind: PhantomData,
         }
     }
+
+    pub fn into_unchecked(self) -> Settings<Unchecked> {
+        let Self {
```

Comment:
wtf is this syntax

Reply:
that was the least amount of effort for me and my vim :laughing: